package-system/python/linux_x64/make-python.sh

```diff
@@ -72,60 +72,78 @@ cp -f -v libexpat/expat/lib/*.c cpython/Modules/expat/
 
 
 echo ""
-echo "--------------- Cloning bzip2 1.0.8 and building ---------------"
+echo "--------------- Cloning libffi 3.4.2 and building static version ---------------"
 echo ""
-git clone git://sourceware.org/git/bzip2.git --branch "bzip2-1.0.8" --depth 1
-if [[ ! -d "bzip2" ]]; then
-    echo "Was unable to create bzip2 dir via git clone.  Is git installed?"
+git clone https://github.com/libffi/libffi.git --branch "v3.4.2" --depth 1
+if [[ ! -d "libffi" ]]; then
+    echo "Was unable to create libffi dir via git clone."
```

Comment:
We still need to build libfii?

Reply:
Yes we do, that hasn't changed. Its passed into configure options for cpython below this line